tools/flecstan/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# -*- CMake -*-
+
+project(flecstan LANGUAGES CXX)
+cmake_minimum_required(VERSION 3.8)
+
+# Use cmake modules from cinch
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../cinch/cmake)
+
+# C++
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pedantic -Wall")
```

Comment:
There two lines should go inside a `if(NOT CMAKE_CXX_FLAGS)` block, if the user wants to e.g. compile with `-O99` we should overwrite his flags.

Reply:
Done. Will upload later today with other changes.